test/cli/exec/run

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -eux
+
+absolute_path() {
+  echo "$(cd "$(dirname "$1")" && pwd)/$(basename "$1")"
+}
+
+MAFIA=$(absolute_path ${1:-./dist/build/mafia/mafia})
+
+MAFIA_TEMP=$(mktemp -d 2>/dev/null || mktemp -d -t 'mafia')
+trap "rm -rf \"${MAFIA_TEMP}\"" EXIT
+
+cd ${MAFIA_TEMP}
+
+MAFIA_HOME=${MAFIA_TEMP} ${MAFIA} exec -- ghc-pkg list base
```

Comment:
Hmm, yeah, I don't mind if you just want to keep it like this, would be nice to actually check it's doing what we want, but I see your point that it's just testing we plumb the right arguments through to cabal.